ide/git/src/org/netbeans/modules/git/ui/repository/RepositoryBrowserPanel.java

```diff
@@ -1226,6 +1227,23 @@ public void run () {
                         });
                     }
                 });
+                actions.add(new AbstractAction(NbBundle.getMessage(RenameBranchAction.class, "LBL_RenameBranchAction_PopupName")) { //NOI18N
+                    @Override
+                    public void actionPerformed (ActionEvent e) {
+                        EventQueue.invokeLater(new Runnable() {
+                            @Override
+                            public void run () {
+                                RenameBranchAction action = SystemAction.get(RenameBranchAction.class);
+                                action.renameBranch(repo, branch);
+                            }
+                        });
+                    }
+
+                    @Override
+                    public boolean isEnabled() {
+                        return !active;
+                    }
```

Comment:
I think this can be removed. It will block renaming the checked out branch in the repository browser:

<img width="415" height="328" alt="Image" src="https://github.com/user-attachments/assets/a792db1c-2ef9-44a2-8ea9-d96082d8673a" />

git allows you to rename the checked out branch without any issues and indeed the option is enabled in the context menu of the project:

<img width="392" height="254" alt="Image" src="https://github.com/user-attachments/assets/83aa026a-ff2b-4f2e-8a70-06831a87741b" />

Reply:
Will check, thx.